### PR TITLE
Do not add runs with nans to the erna database

### PR DESCRIPTION
**Pull request by MaxNoe**

### CODE REVIEW

File: erna/automatic_processing/database_utils.py [hunk 1]
```diff
@@ -182,11 +181,18 @@ def insert_new_job(
 @requires_database_connection
 def insert_new_jobs(raw_data_files, jar, xml, queue, progress=True, **kwargs):
 
+    failed_files = []
     for f in tqdm(raw_data_files, total=raw_data_files.count(), disable=not progress):
         try:
             insert_new_job(f, jar=jar, xml=xml, queue=queue, **kwargs)
         except peewee.IntegrityError:
-            log.warning('Job already submitted: {} {}'.format(f.night, f.run_id))
+            log.warning('Job already submitted: {}_{:03d}'.format(f.night, f.run_id))
```
Comment (jebuss) [inline erna/automatic_processing/database_utils.py]: why do you limit the number of digits?

File: erna/scripts/fill_database.py [hunk 2]
```diff
@@ -56,9 +56,23 @@ def main(start, end, config):
...
+        print(data_runs[nan_entries])
+        data_runs.dropna(inplace=True)
+
+    fill_data_runs(data_runs, database=database)
+
     # fill all drs runs into drs_files
-    fill_drs_runs(runs.query('(fRunTypeKey == 2) & (fDrsStep == 2)'), database=database)
+    drs_runs = runs.query('(fRunTypeKey == 2) & (fDrsStep == 2)')
+    nan_entries = drs_runs.isnull().any(axis=1)
+    if len(drs_runs[nan_entries]) != 0:
```
Comment (jebuss) [inline erna/scripts/fill_database.py]: can you avoid this code duplication?

File: erna/automatic_processing/database_utils.py (hunk 1 above)
Comment (MaxNoe) [reply erna/automatic_processing/database_utils.py]: I don't limit the number of digits, i zero fill it (and we never have more than 999 runs per night)

File: erna/scripts/fill_database.py (hunk 2 above)
Comment (MaxNoe) [reply erna/scripts/fill_database.py]: no worth it in my opinion

File: erna/automatic_processing/database_utils.py (hunk 1 above)
Comment (jebuss) [reply erna/automatic_processing/database_utils.py]: I see

File: erna/scripts/fill_database.py (hunk 2 above)
Comment (jebuss) [reply erna/scripts/fill_database.py]: fine with me

**Pull request merged**